astropy/visualization/zscale.py

```diff
@@ -0,0 +1,106 @@
+# Licensed under a 3-clause BSD style license - see NUMDISPLAY_LICENSE.rst
+
+"""
+Zscale implementation based on the one from the STScI Numdisplay package.
+
+Original implementation from Numdisplay is BSD licensed:
+https://trac.stsci.edu/ssb/stsci_python/browser/stsci_python/trunk/numdisplay/lib/stsci/numdisplay/zscale.py?rev=19347
+https://trac.stsci.edu/ssb/stsci_python/browser/stsci_python/trunk/numdisplay/LICENSE.txt?rev=19347
+
+"""
+
+from __future__ import absolute_import, division
+
+import numpy as np
+
+__all__ = ['zscale']
+
+MAX_REJECT = 0.5
+MIN_NPIXELS = 5
+KREJ = 2.5
+MAX_ITERATIONS = 5
```

Comment:
I think it's better to remove these module-level constants and instead make them (optional) keywords in the functions.